src/Entity/Server.php

```diff
@@ -123,6 +129,27 @@ class Server extends ConfigEntityBase implements ServerInterface {
    */
   public $batching = TRUE;
 
+  /**
+   * Whether to disable query introspection.
+   *
+   * @var bool
+   */
+  public $disable_introspection = FALSE;
+
+  /**
+   * The query complexity.
```

Comment:
Should be similar to the form description like "The maximum allowed query complexity, NULL means unlimited."